package-lock.json

```diff
@@ -348,6 +350,24 @@
 				}
 			}
 		},
+		"@automattic/notifications": {
+			"version": "file:apps/notifications",
+			"requires": {
```

Comment:
I think `apps/notifications` also need `@automattic/calypso-build`. We seem to have forgotten to add this to the app `package.json` file.

I think this applies to more instances too, for example `@automattic/calypso-color-schemes` also seems to use `@automattic/calypso-build`.

Reply:
Definitely, many packages/apps are missing some dependencies and/or devDependencies.

My plan was to tackle those individually to avoid unintended changes to the dependency tree (like https://github.com/Automattic/wp-calypso/pull/40054).

Do you think those should be done in this PR?

Reply:
We can totally do it in another PR 👍

Reply:
Keep in mind that if things were done 100% correctly, there should be two Notifications modules:
- the package that can be bundled with Calypso or any other app. Calypso now imports it as `import '../../../apps/notifications/index.jsx'` instead of depending on a package
- standalone app that wraps the package in a thin "shell" and is loaded as a script into, e.g., site frontend.

In that light, using the `@automattic/notifications` name for the app is not a good idea -- it should be the name of the package instead.